sample/breakpoints.css.scss

```diff
@@ -0,0 +1,85 @@
+
+@import '../grid-system.scss';
+
+.app {
+	@include vui-gs-container;
+}
+
+.app * {
+	box-sizing: border-box;
+}
+
+.header {
+	@include vui-gs-row;
+	background-color: rgba(0,0,250,0.4);
+	height: 60px;
+	text-align: center;
+}
+
+.stuff-1 > div {
+	@include vui-gs-span(12 break);
+	background-color: rgba(250,0,0,0.4);
+	height: 60px;
+}
+
+#bp-extra-small {
+	display: block;
+}
+
+#bp-small {
+	display: none;
+}
+
+#bp-medium {
+	display: none;
+}
+
+@include vui-gs-small() {
+
+	#bp-extra-small {
+		display: none;
+	}
+
+	#bp-small {
+		display: block;
+	}
+
+	#bp-medium {
+		display: none;
+	}
+
+	.stuff-1 > div {
+		@include vui-gs-span(4 nobreak);
```

Comment:
Since no max-width constraint is applied in small, this `nobreak` is required to ensure these spans float.